prompts/self-ossinstruct-fewshot.txt

```diff
@@ -681,4 +681,83 @@ assert create_folds(range(12), 2) == [range(0, 6), range(6, 12)]
 assert create_folds(range(8), 2) == [range(0, 4), range(4, 8)]
 assert create_folds(range(25), -5) == []
 assert create_folds(range(6), 2) == [range(0, 3), range(3, 6)]
-```
\ No newline at end of file
+```
+
+### Example 10
+[Code]
+import math
+
+def alt2temp(h, mmHg=29.92):
+    if mmHg != 29.92:
+        dh = ((8.95e10 * 288.15) /
+              (-32.174 * 28.964) *
+              math.log(mmHg / 29.92))
+        h += dh
+    To = 518.69
+    if h > 36089:
+        T = 389.99
+    else:
+        T = To * (1 - 6.87559e-6 * h)
+    return T
+
+[Property]
+category: code generation (function implementation)
+language: Python
+concepts: mathematical modeling, environmental physics, parameter default values
```

Comment:
mathematical modeling and environmental physics are too vague concepts. Try to relate to the logic like formula calculation / logarithm using..